packages/xstate-inspect/test/inspect.test.ts

```diff
@@ -141,4 +141,54 @@ describe('@xstate/inspect', () => {
       }
     });
   });
+
+  it('should handle using serializer to remove dangerous values', (done) => {
+    const circularStructure = {} as any;
+    let deepRef = circularStructure;
+    for (let i = 0; i < 1_000_000; i += 1) {
```

Comment:
The actual depth errors start to occur is somewhere around about 5,275 on my machine. Not sure that that value would stay the same so I picked a really big one.

Reply:
Did redo this back down to 20,000 so it's a bit faster but still shows an error.